app/scripts/modules/amazon/src/function/configure/FunctionTags.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+import { IWizardPageComponent, MapEditor, FormikFormField, FormValidator } from '@spinnaker/core';
+import { FormikProps } from 'formik';
+import { IAmazonFunctionUpsertCommand } from 'amazon/index';
+import { IAmazonFunction } from 'amazon/domain';
+import * as _ from 'lodash';
+import { awsTagsValidator } from 'amazon/aws.validators';
+
+export interface IFunctionTagsProps {
+  formik: FormikProps<IAmazonFunctionUpsertCommand>;
+  isNew?: boolean;
+  functionDef: IAmazonFunction;
+}
+
+export class FunctionTags extends React.Component<IFunctionTagsProps>
+  implements IWizardPageComponent<IAmazonFunctionUpsertCommand> {
+  constructor(props: IFunctionTagsProps) {
+    super(props);
+  }
+  public validate = (values: IAmazonFunctionUpsertCommand) => {
+    const validator = new FormValidator(values);
+    validator
+      .field('tags', 'Tag')
+      .required()
+      .withValidators(awsTagsValidator);
+    return validator.validateForm();
+  };
+
+  private varsChanged = (tag: string | { [key: string]: string }) => {
+    this.props.formik.setFieldValue('tags', Array(tag));
+  };
+
+  public render() {
+    const { values } = this.props.formik;
+    return (
+      <div className="container-fluid form-horizontal ">
+        <FormikFormField
+          fastField={false}
+          name="tags"
+          input={props => <MapEditor {...props} model={values.tags} allowEmpty={false} onChange={this.varsChanged} />}
```

Comment:
I'm wondering if this one was missed, or if there is some reason you can't use `MapEditorInput`?

Reply:
Sorry I missed this one.
Changed it to MapEditorInput.